src/filesinkhelper.ipp

```diff
@@ -51,7 +51,7 @@ namespace g3 {
          return prefix;
       }
 
-      std::string pathSanityFix(std::string path, std::string file_name) {
```

Comment:
nit, but good change. It's within single-threaded scope and only used internally